sky/backends/backend_utils.py

```diff
@@ -799,28 +801,31 @@ def write_cluster_config(
                 'ssh_private_key': (None if auth_config is None else
                                     auth_config['ssh_private_key']),
             }),
-    )
+        output_path=tmp_yaml_path)
     config_dict['cluster_name'] = cluster_name
     config_dict['ray'] = yaml_path
     if dryrun:
         return config_dict
```

Comment:
Should we return the old `yaml_path` for dryrun? I would prefer we return the `tmp_yaml_path`, so that the caller can get the correct yaml content.

Reply:
Just fixed!